staging/src/kubevirt.io/api/instancetype/v1alpha1/types.go

```diff
@@ -161,7 +161,7 @@ type CPUInstancetype struct {
 type MemoryInstancetype struct {
 
 	// Required amount of memory which is visible inside the guest OS.
-	Guest *resource.Quantity `json:"guest,omitempty"`
+	Guest *resource.Quantity `json:"guest"`
```

Comment:
This field doesn't need to be a pointer, because it is required.

Reply:
ACK fair point, I'll rework shortly.